cmd/permission/main.go

```diff
@@ -2,24 +2,23 @@ package main
 
 import (
 	"fmt"
+
+	"github.com/openslides/openslides-permission-service/internal/permission"
```

Comment:
You have to import the other package. In my environment this is automatically done by vscode which uses [gopls](https://github.com/golang/tools/blob/master/gopls/README.md) in the background. 

You should also change the package name from `github.com/openslides/...` to `github.com/OpenSlides/...`

Github is case sensitive but go is not. So github.com/OpenSlides/openslides-permission-service is a different package. I had problems with this in the past.